### DISCUSSION TITLE
11.0.0 Preview 6 - AvaloniaProperty.Register - Without notifying-parameter?

### DISCUSSION THREAD

**Discussion by Maruhl** [AvaloniaUI/Avalonia#10766] [Q&A] [answered]
Hi, simple question.

After updating from prev 5 to prev 6, the parameter for notifying a change for the value is suddenly no longer available.
![grafik](https://user-images.githubusercontent.com/41585619/227272191-44caa3bf-fa86-40a0-9f7e-419e834e292e.png)

I see in the code that this method is now internal, is there a reason for this? Or is there another way to achieve the same functionality now?

![grafik](https://user-images.githubusercontent.com/41585619/227272855-329022d9-2ec6-410a-b314-023ef77be910.png)

**Comment by jp2masa** [comment, marked as answer]
https://github.com/AvaloniaUI/Avalonia/issues/3523#issuecomment-582456173
https://github.com/AvaloniaUI/Avalonia/issues/10088#issuecomment-1428541052
